data/gm/functions/zzz/modulo/divide.mcfunction

```diff
@@ -0,0 +1,16 @@
+#> gm:zzz/modulo/divide
+#
+# Divides var2 by var3
+#
+# ---
+# @context any
+# @within gm:modulo
+
+say divide
```

Comment:
:)

Reply:
already removed the say don't worry about it https://github.com/gibbsly/gm/commit/e6efebb2b2762a75c8f7da5bb8c75fc1d3034930